_data/events.json

```diff
@@ -0,0 +1,321 @@
+      "description": "16th biennial conference of the Association for Machine Translation in the Americas",
+      "startDate":"2024-09-30",
+      "endDate":"2024-10-02",
+      "location":{
+         "type":"VirtualLocation",
+         "name":"Online",
+         "eventAttendanceMode": "OnlineEventAttendanceMode"
+      },
+      "organizer":{
+         "type":"Organization",
+         "name":"Association for Machine Translation in the Americas",
+         "url":"https://amtaweb.org"
+      },
+      "futureTenseOpeningParagraph": "The sixteenth Biennial Conference of the Association for Machine Translation in the Americas (<strong><a href='/amta'>AMTA</a> 2024</strong>) will be hosted in Chicago, Illinois from 30 September to 2 October, 2024. <br><br> Workshops and tutorials will take place earlier, mid-September. The date is yet to be announced.",
+      "pastTenseOpeningParagraph": "The sixteenth Biennial Conference of the Association for Machine Translation in the Americas (<strong><a href='/amta'>AMTA</a> 2024</strong>) was hosted in Chicago, Illinois from 30 September to 2 October, 2024.",
+      "importantDates": [
+         {
+            "name": "Submission deadline",
+            "date": "6 June "
+         },
+         {
+            "name": "Notification of acceptance",
+            "date": "18 July "
+         },
+         {
+            "name": "Final “camera-ready” papers for proceedings",
+            "date": "16 August"
+         }
+      ]
+   },
+   {
+      "name":"AmericasNLP 2024",
+      "id": "americasnlp-2024",
+      "description": "Machine translation for indigenous languages of the Americas",
+      "startDate":"2024-06-20",
+      "endDate":"2024-06-21",
+      "location":{
+         "type":"Place",
+         "name":"Mexico City, Mexico",
+         "eventAttendanceMode": "OfflineEventAttendanceMode"
+      },
+      "organizer":{
+         "type":"Person",
+         "url":"http://turing.iimas.unam.mx/americasnlp/st.html"
+      },
+      "futureTenseOpeningParagraph": "The <strong>Fourth AmericasNLP Competition</strong> will take place online in June, at NAACL 2024 in Mexico City. The competition focused on creating machine translation systems for indigenous languages from the Americas.",
+      "pastTenseOpeningParagraph": "The <strong>Fourth AmericasNLP Competition</strong> took place online in June, at NAACL 2024 in Mexico City. The competition focused on creating machine translation systems for indigenous languages from the Americas.",
+      "links": [
+         "<a href='http://turing.iimas.unam.mx/americasnlp/2024_workshop.html'>turing.iimas.unam.mx/americasnlp/2024_workshop.html</a>"
+      ],
+      "sharedTasks": [
+         "A machine translation shared task on truly low-resource languages.",
+         "A shared task on morphological adaptation to generate educational examples."
+      ],
+      "speakers": [
+         {
+            "type": "Keynote speakers",
+            "about": [
```

Comment:
feels wrong

i'd expect something more like this:

```
"speakers": [
       { name: "Graham Neubig", type: "keynote" },
       { name: "Jaime Pérez González", type: "keynote" },
       "Manuel Mager",
       "Abteen Ebrahimi",
       "Shruti Rijhwani",
       "Arturo Oncevay",
       "Luis Chiruzzo",
       "Robert Pugh",
       "Katharina Kann"
]
```

though not sure if the schema will allow either string or object

Reply:
The schema won't allow this since an array should contain elements of the same type.

Probably something like this can do:

```"speakers": [
    { "name": "Graham Neubig", "type": "keynote" },
    { "name": "Jaime Pérez González", "type": "keynote" },
    { "name": "Manuel Mager" },
    { "name": "Abteen Ebrahimi" },
    { "name": "Shruti Rijhwani" },
    { "name": "Arturo Oncevay" },
    { "name": "Luis Chiruzzo" },
    { "name": "Robert Pugh" },
    { "name": "Katharina Kann" }
]```

@bittlingmayer, What do you think?